packages/widgets/src/tabbar.ts

```diff
@@ -1830,7 +1830,8 @@ namespace Private {
     let node = document.createElement('div');
     let content = document.createElement('ul');
     content.setAttribute('role', 'tablist');
-    content.className = 'lm-TabBar-content';
+    content.setAttribute('tabindex', '0');
```

Comment:
Please provide a rationale for setting the tabindex in the PR description explaining why it is needed and linking any relevant issues (in this or other repositories).